src/components/ModelingSidebar/ModelingSidebar.tsx

```diff
@@ -41,7 +42,7 @@ export function ModelingSidebar({ paneOpacity }: ModelingSidebarProps) {
   const onboardingStatus = settings.context.app.onboardingStatus
   const { send, context } = useModelingContext()
   const pointerEventsCssClass =
-    onboardingStatus.current === 'camera' ||
+    onboardingStatus.current === onboardingPaths.CAMERA ||
```

Comment:
Is it safe to say that `onboardingStatus.current` will now be of the format `/camera`? 

I looked at the constant you have here 
```
CAMERA: '/camera',
```

but the old code says `'camera'`. Is this a bug or the new expected constant?

Reply:
that was a bug

Reply:
the types actually check now :) there was lots of loose holes here

Reply:
perfect